packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -479,6 +471,8 @@ addGraphQLResolvers({
             title: docMetadata.name,
             contents: {
               originalContents,
+              commitMessage,
+              googleDocMetadata: docMetadata
```

Comment:
Unrelated: Adding these fields for the case where a post is being created rather than edited. The main benefit of this is that it will make counting posts that have been imported from google docs much easier (also the commit message shows in the version history editor)